src/date.ts

```diff
@@ -1,6 +1,22 @@
 import type { Faker } from '.';
 import type { DateEntryDefinition } from './definitions';
 
+const toDate = (date?: string | Date): Date => {
+  if (date != null) {
+    return new Date(date instanceof Date ? date : Date.parse(date));
+  }
+
+  return new Date();
+};
```

Comment:
On other places we already used the code-style using functions instead of arrow functions, so lets do here too

```suggestion
function toDate(date?: string | Date): Date {
  if (date != null) {
    return new Date(date instanceof Date ? date : Date.parse(date));
  }

  return new Date();
}
```

Reply:
Not a major issue, but sure, why not...